system-contracts/contracts/L1Messenger.sol

```diff
@@ -209,7 +209,7 @@ contract L1Messenger is IL1Messenger, ISystemContract {
         calldataPtr += 4;
 
         bytes32[] memory l2ToL1LogsTreeArray = new bytes32[](L2_TO_L1_LOGS_MERKLE_TREE_LEAVES);
-        bytes32 reconstructedChainedLogsHash;
+        bytes32 reconstructedChainedLogsHash = bytes32(0);
```

Comment:
Why?

Reply:
for all these OZ suggested adding in the default values. i dont have a strong opinion on keeping or removing